src/agent/Cargo.toml

```
@@ -84,6 +84,7 @@ regorus = { version = "0.1.4", default-features = false, features = [
     "arc",
     "regex",
 ], optional = true }
+cdi = { git = "https://github.com/zvonkok/container-device-interface-rs", branch = "main" }
```

As cdi-rs has been transferred to CNCF-Tags, IMO, it should replaced now !
```suggestion
cdi = { git = "https://github.com/cncf-tags/container-device-interface-rs", branch = "main" }
```

---

Definitely!